configs/tsconfig-compass/tsconfig.common.json

```diff
@@ -1,3 +1,6 @@
 {
-  "extends": "@mongodb-js/tsconfig-devtools/tsconfig.common.json"
+  "extends": "@mongodb-js/tsconfig-devtools/tsconfig.common.json",
+  "compilerOptions": {
+    "moduleResolution": "node16"
+  }
```

Comment:
... so that the `'@mongodb-js/devtools-proxy-support/proxy-options'` import works (which in turn also causes the small change to the `mongodb-explain-compat` package.json).

Reply:
Woah, last time I tried to turn it on, a bunch of other things broke, which are maybe gone now? This means that we can also remove weird workarounds for provider re-exports, awesome!

Reply:
Fwiw, `npm run bootstrap` passed just fine – but this failed during the e2e test phase. Probably not hard to fix if we want to, though